[mpg123] missing export symbols such as `mpg123_open`

[Upstream report](https://sourceforge.net/p/mpg123/bugs/374/)

Unsure where the fault lies at the moment but a workaround to get the symbols I missed was to downgrade to `1.29.3 0d8db63f9bae1507887bc5241a56abccfeb767b7ba8362eb0fce9de2f63369e57fdd6f25a953f8ef5f9ead4f400237db51914816e278566fdf8e6f205ebca5d6` and removing the now-conflicting FPU patch.

Neither the current version 1.32.7 nor latest 1.32.9 work.

The resulting library does not export the symbols it should, here it is missing `mpg123_open`:

```powershell
&"C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\14.42.34433\bin\Hostx64\x64\dumpbin.exe" /EXPORTS C:\Users\Yeet\Tauon\vcpkg\installed\x64-windows\lib\mpg123.lib
...
                  mpg123_new_string
                  mpg123_open_feed
                  mpg123_open_handle
                  mpg123_outblock
                  mpg123_par
...
```

<details><summary>Debug build log + symbol check</summary>

```diff
PS C:\Users\Yeet\Tauon\vcpkg> git diff
diff --git a/ports/mpg123/portfile.cmake b/ports/mpg123/portfile.cmake
index 26c2074..4cffbfc 100644
--- a/ports/mpg123/portfile.cmake
+++ b/ports/mpg123/portfile.cmake
@@ -3,7 +3,7 @@ vcpkg_from_sourceforge(
     REPO mpg123/mpg123
     REF "${VERSION}"
     FILENAME "mpg123-${VERSION}.tar.bz2"
-    SHA512 71f7cf6035c489934400528d0eaf0c1104417867990dd0abcc99c682818ef1f3e99dbee9dcdd888d88ed172a8950d5df515b755a5c1c1b54fc56a28584ceef8c
+    SHA512 dccb640b0845061811cb41bf145587e7093556d686d49a748232b079659b46284b6cc40db42d14febceac11277c58edf2b69d1b4c46c223829a3d15478e2e26c
     PATCHES
         have-fpu.diff
 )
diff --git a/ports/mpg123/vcpkg.json b/ports/mpg123/vcpkg.json
index db61543..07e023a 100644
--- a/ports/mpg123/vcpkg.json
+++ b/ports/mpg123/vcpkg.json
@@ -1,6 +1,6 @@
 {
   "name": "mpg123",
-  "version": "1.32.7",
+  "version": "1.32.9",
   "description": "mpg123 is a real time MPEG 1.0/2.0/2.5 audio player/decoder for layers 1, 2 and 3 (MPEG 1.0 layer 3 also known as MP3).",
   "homepage": "https://sourceforge.net/projects/mpg123/",
   "license": "LGPL-2.1-or-later",
```

```powershell
PS C:\Users\Yeet\Tauon\vcpkg> ./vcpkg.exe remove mpg123:x64-windows --recurse
The following packages will be removed:
    mpg123:x64-windows
Removing 1/1 mpg123:x64-windows
PS C:\Users\Yeet\Tauon\vcpkg> &"C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\14.42.34433\bin\Hostx64\x64\dumpbin.exe" /EXPORTS C:\Users\Yeet\Tauon\vcpkg\installed\x64-windows\lib\mpg123.lib
Microsoft (R) COFF/PE Dumper Version 14.42.34433.0
Copyright (C) Microsoft Corporation.  All rights reserved.


Dump of file C:\Users\Yeet\Tauon\vcpkg\installed\x64-windows\lib\mpg123.lib
LINK : fatal error LNK1181: cannot open input file 'C:\Users\Yeet\Tauon\vcpkg\installed\x64-windows\lib\mpg123.lib'
```

```powershell
PS C:\Users\Yeet\Tauon\vcpkg> ./vcpkg.exe install mpg123:x64-windows --debug
[DEBUG] To include the environment variables in debug output, pass --debug-env
[DEBUG] Trying to load bundleconfig from C:\Users\Yeet\Tauon\vcpkg\vcpkg-bundle.json
[DEBUG] Failed to open: C:\Users\Yeet\Tauon\vcpkg\vcpkg-bundle.json
[DEBUG] Bundle config: readonly=false, usegitregistry=false, embeddedsha=nullopt, deployment=Git, vsversion=nullopt
[DEBUG] Metrics enabled.
[DEBUG] Feature flag 'binarycaching' unset
[DEBUG] Feature flag 'compilertracking' unset
[DEBUG] Feature flag 'registries' unset
[DEBUG] Feature flag 'versions' unset
[DEBUG] Feature flag 'dependencygraph' unset
[DEBUG] Using scripts-root: C:\Users\Yeet\Tauon\vcpkg\scripts
[DEBUG] Using builtin-ports: C:\Users\Yeet\Tauon\vcpkg\ports
[DEBUG] Using installed-root: C:\Users\Yeet\Tauon\vcpkg\installed
[DEBUG] Using buildtrees-root: C:\Users\Yeet\Tauon\vcpkg\buildtrees
[DEBUG] Using packages-root: C:\Users\Yeet\Tauon\vcpkg\packages
[DEBUG] Using vcpkg-root: C:\Users\Yeet\Tauon\vcpkg
[DEBUG] Using builtin-registry: C:\Users\Yeet\Tauon\vcpkg\versions
[DEBUG] Using downloads-root: C:\Users\Yeet\Tauon\vcpkg\downloads
[DEBUG] External asset downloads are allowed (x-block-origin is disabled)...
[DEBUG] Asset cache is not configured.
[DEBUG] Detecting host with IsWow64Process2
[DEBUG] Detecting host with IsWow64Process2
Computing installation plan...
[DEBUG] Loading dep info for: mpg123:x64-windows
[DEBUG] 1000: CreateProcessW("C:\Users\Yeet\Tauon\vcpkg\downloads\tools\cmake-3.30.1-windows\cmake-3.30.1-windows-i386/bin/cmake.exe" --version)
[DEBUG] 1000: cmd_execute_and_stream_data() returned 0 after   266778 us
[DEBUG] 1001: CreateProcessW("C:\Users\Yeet\Tauon\vcpkg\downloads\tools\cmake-3.30.1-windows\cmake-3.30.1-windows-i386/bin/cmake.exe" -DVCPKG_ROOT_DIR=C:/Users/Yeet/Tauon/vcpkg -DPACKAGES_DIR=C:/Users/Yeet/Tauon/vcpkg/packages -DBUILDTREES_DIR=C:/Users/Yeet/Tauon/vcpkg/buildtrees -D_VCPKG_INSTALLED_DIR=C:/Users/Yeet/Tauon/vcpkg/installed -DDOWNLOADS=C:/Users/Yeet/Tauon/vcpkg/downloads -DVCPKG_MANIFEST_INSTALL=OFF -P "C:\Users\Yeet\Tauon\vcpkg\buildtrees\0.vcpkg_dep_info.cmake")
[DEBUG] 1001: cmd_execute_and_stream_data() returned 0 after    68629 us
[DEBUG] Loading dep info for: yasm:x64-windows
[DEBUG] 1002: CreateProcessW("C:\Users\Yeet\Tauon\vcpkg\downloads\tools\cmake-3.30.1-windows\cmake-3.30.1-windows-i386/bin/cmake.exe" -DVCPKG_ROOT_DIR=C:/Users/Yeet/Tauon/vcpkg -DPACKAGES_DIR=C:/Users/Yeet/Tauon/vcpkg/packages -DBUILDTREES_DIR=C:/Users/Yeet/Tauon/vcpkg/buildtrees -D_VCPKG_INSTALLED_DIR=C:/Users/Yeet/Tauon/vcpkg/installed -DDOWNLOADS=C:/Users/Yeet/Tauon/vcpkg/downloads -DVCPKG_MANIFEST_INSTALL=OFF -P "C:\Users\Yeet\Tauon\vcpkg\buildtrees\1.vcpkg_dep_info.cmake")
[DEBUG] 1002: cmd_execute_and_stream_data() returned 0 after    82288 us
[DEBUG] 1003: CreateProcessW("C:\Users\Yeet\Tauon\vcpkg\downloads\tools\cmake-3.30.1-windows\cmake-3.30.1-windows-i386/bin/cmake.exe" -DVCPKG_ROOT_DIR=C:/Users/Yeet/Tauon/vcpkg -DPACKAGES_DIR=C:/Users/Yeet/Tauon/vcpkg/packages -DBUILDTREES_DIR=C:/Users/Yeet/Tauon/vcpkg/buildtrees -D_VCPKG_INSTALLED_DIR=C:/Users/Yeet/Tauon/vcpkg/installed -DDOWNLOADS=C:/Users/Yeet/Tauon/vcpkg/downloads -DVCPKG_MANIFEST_INSTALL=OFF -P "C:\Users\Yeet\Tauon\vcpkg\buildtrees\0.vcpkg_tags.cmake")
[DEBUG] 1003: cmd_execute_and_stream_data() returned 0 after   119769 us
The following packages will be built and installed:
    mpg123:x64-windows@1.32.9
[DEBUG] 1004: CreateProcessW("C:\Program Files (x86)\Microsoft Visual Studio\Installer\vswhere.exe" -all -prerelease -legacy -products * -format xml)
[DEBUG] 1004: cmd_execute_and_stream_data() returned 0 after    73618 us
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\toolchains/windows.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\toolchains/windows.cmake has hash 0433e2303d8e9448dd9cf29d3052811cb89f6324bfd45703cfd4ec1e4134228d
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\triplets\x64-windows.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\triplets\x64-windows.cmake has hash 4556164a2cd3dd6f4742101eabb46def7e71b6e5856faa88e5d005aac12a803c
Detecting compiler hash for triplet x64-windows...
[DEBUG] Found path: C:\Program Files\Git\cmd\git.exe
[DEBUG] 1005: CreateProcessW("C:\Program Files\Git\cmd\git.exe" --version)
[DEBUG] 1005: cmd_execute_and_stream_data() returned 0 after   107246 us
[DEBUG] Detecting host with IsWow64Process2
[DEBUG] Found path: C:\Program Files\PowerShell\7\pwsh.exe
[DEBUG] 1006: CreateProcessW("C:\Program Files\PowerShell\7\pwsh.exe" --version)
[DEBUG] 1006: cmd_execute_and_stream_data() returned 0 after   233341 us
[DEBUG] command line: cmd /d /c "C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Auxiliary/Build\vcvarsall.bat" -vcvars_ver=14.42.34433 amd64   2>&1 <NUL  & echo cdARN4xjKueKScMy9C6H & set
[DEBUG] 1007: CreateProcessW(cmd /d /c "C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Auxiliary/Build\vcvarsall.bat" -vcvars_ver=14.42.34433 amd64   2>&1 <NUL  & echo cdARN4xjKueKScMy9C6H & set)
[DEBUG] 1007: cmd_execute_and_stream_data() returned 0 after  2031462 us
[DEBUG] **********************************************************************
** Visual Studio 2022 Developer Command Prompt v17.12.1
** Copyright (c) 2022 Microsoft Corporation
**********************************************************************
[vcvarsall.bat] Environment initialized for: 'x64'
cdARN4xjKueKScMy9C6H
ALLUSERSPROFILE=C:\ProgramData
APPDATA=C:\Users\Yeet\AppData\Roaming
CommandPromptType=Native
CommonProgramFiles=C:\Program Files\Common Files
CommonProgramFiles(x86)=C:\Program Files (x86)\Common Files
CommonProgramW6432=C:\Program Files\Common Files
COMPUTERNAME=DESKTOP-QUNJNVR
ComSpec=C:\WINDOWS\system32\cmd.exe
DevEnvDir=C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\Common7\IDE\
ExtensionSdkDir=C:\Program Files (x86)\Microsoft SDKs\Windows Kits\10\ExtensionSDKs
EXTERNAL_INCLUDE=C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\14.42.34433\include;C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Auxiliary\VS\include;C:\Program Files (x86)\Windows Kits\10\include\10.0.22621.0\ucrt;C:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\um;C:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\shared;C:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\winrt;C:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\cppwinrt;C:\Program Files (x86)\Windows Kits\NETFXSDK\4.8\include\um
Framework40Version=v4.0
FrameworkDir=C:\Windows\Microsoft.NET\Framework64\
FrameworkDir64=C:\Windows\Microsoft.NET\Framework64\
FrameworkVersion=v4.0.30319
FrameworkVersion64=v4.0.30319
HOMEDRIVE=C:
HOMEPATH=\Users\Yeet
INCLUDE=C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\14.42.34433\include;C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Auxiliary\VS\include;C:\Program Files (x86)\Windows Kits\10\include\10.0.22621.0\ucrt;C:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\um;C:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\shared;C:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\winrt;C:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\cppwinrt;C:\Program Files (x86)\Windows Kits\NETFXSDK\4.8\include\um
LIB=C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\14.42.34433\lib\x64;C:\Program Files (x86)\Windows Kits\NETFXSDK\4.8\lib\um\x64;C:\Program Files (x86)\Windows Kits\10\lib\10.0.22621.0\ucrt\x64;C:\Program Files (x86)\Windows Kits\10\\lib\10.0.22621.0\\um\x64
LIBPATH=C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\14.42.34433\lib\x64;C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\14.42.34433\lib\x86\store\references;C:\Program Files (x86)\Windows Kits\10\UnionMetadata\10.0.22621.0;C:\Program Files (x86)\Windows Kits\10\References\10.0.22621.0;C:\Windows\Microsoft.NET\Framework64\v4.0.30319
LOCALAPPDATA=C:\Users\Yeet\AppData\Local
LOGONSERVER=\\DESKTOP-QUNJNVR
NETFXSDKDir=C:\Program Files (x86)\Windows Kits\NETFXSDK\4.8\
NUMBER_OF_PROCESSORS=2
OS=Windows_NT
PATHEXT=.COM;.EXE;.BAT;.CMD;.VBS;.VBE;.JS;.JSE;.WSF;.WSH;.MSC;.PY;.PYW;.CPL
Platform=x64
PROCESSOR_ARCHITECTURE=AMD64
PROCESSOR_IDENTIFIER=AMD64 Family 25 Model 97 Stepping 2, AuthenticAMD
PROCESSOR_LEVEL=25
PROCESSOR_REVISION=6102
ProgramData=C:\ProgramData
ProgramFiles=C:\Program Files
ProgramFiles(x86)=C:\Program Files (x86)
ProgramW6432=C:\Program Files
PROMPT=$P$G
PSModulePath=C:\Users\Yeet\Documents\WindowsPowerShell\Modules;C:\Program Files\WindowsPowerShell\Modules;C:\WINDOWS\system32\WindowsPowerShell\v1.0\Modules
PUBLIC=C:\Users\Public
SystemDrive=C:
SystemRoot=C:\WINDOWS
TEMP=C:\Users\Yeet\AppData\Local\Temp
TMP=C:\Users\Yeet\AppData\Local\Temp
UCRTVersion=10.0.22621.0
UniversalCRTSdkDir=C:\Program Files (x86)\Windows Kits\10\
USERDOMAIN=DESKTOP-QUNJNVR
USERDOMAIN_ROAMINGPROFILE=DESKTOP-QUNJNVR
USERNAME=Yeet
USERPROFILE=C:\Users\Yeet
VCIDEInstallDir=C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\Common7\IDE\VC\
VCINSTALLDIR=C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\
VCToolsInstallDir=C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\14.42.34433\
VCToolsRedistDir=C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Redist\MSVC\14.42.34433\
VCToolsVersion=14.42.34433
VisualStudioVersion=17.0
VS170COMNTOOLS=C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\Common7\Tools\
VSCMD_ARG_app_plat=Desktop
VSCMD_ARG_HOST_ARCH=x64
VSCMD_ARG_TGT_ARCH=x64
VSCMD_ARG_VCVARS_VER=14.42.34433
VSCMD_VER=17.12.1
VSINSTALLDIR=C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\
windir=C:\WINDOWS
PATH=C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\14.42.34433\bin\HostX64\x64;C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\Common7\IDE\VC\VCPackages;C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\Common7\IDE\CommonExtensions\Microsoft\TestWindow;C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\MSBuild\Current\bin\Roslyn;C:\Program Files (x86)\Microsoft SDKs\Windows\v10.0A\bin\NETFX 4.8 Tools\x64\;C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\Common7\IDE\Extensions\Microsoft\CodeCoverage.Console;C:\Program Files (x86)\Windows Kits\10\bin\10.0.22621.0\\x64;C:\Program Files (x86)\Windows Kits\10\bin\\x64;C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\\MSBuild\Current\Bin\amd64;C:\Windows\Microsoft.NET\Framework64\v4.0.30319;C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\Common7\IDE\;C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\Common7\Tools\;C:\Program Files\PowerShell\7;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\system32\Wbem;C:\WINDOWS\system32\WindowsPowerShell\v1.0\;C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin;C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\Common7\IDE\CommonExtensions\Microsoft\CMake\Ninja;C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\Common7\IDE\VC\Linux\bin\ConnectionManagerExe
VSLANG=1033
VSCMD_SKIP_SENDTELEMETRY=1
VCPKG_COMMAND=C:/Users/Yeet/Tauon/vcpkg/vcpkg.exe
WindowsLibPath=C:\Program Files (x86)\Windows Kits\10\UnionMetadata\10.0.22621.0;C:\Program Files (x86)\Windows Kits\10\References\10.0.22621.0
WindowsSdkBinPath=C:\Program Files (x86)\Windows Kits\10\bin\
WindowsSdkDir=C:\Program Files (x86)\Windows Kits\10\
WindowsSDKLibVersion=10.0.22621.0\
WindowsSdkVerBinPath=C:\Program Files (x86)\Windows Kits\10\bin\10.0.22621.0\
WindowsSDKVersion=10.0.22621.0\
WindowsSDK_ExecutablePath_x64=C:\Program Files (x86)\Microsoft SDKs\Windows\v10.0A\bin\NETFX 4.8 Tools\x64\
WindowsSDK_ExecutablePath_x86=C:\Program Files (x86)\Microsoft SDKs\Windows\v10.0A\bin\NETFX 4.8 Tools\
X_VCPKG_RECURSIVE_DATA={}

__DOTNET_ADD_64BIT=1
__DOTNET_PREFERRED_BITNESS=64
__VSCMD_PREINIT_PATH=C:\Program Files\PowerShell\7;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\system32\Wbem;C:\WINDOWS\system32\WindowsPowerShell\v1.0\

[DEBUG] 1008: CreateProcessW("C:\Users\Yeet\Tauon\vcpkg\downloads\tools\cmake-3.30.1-windows\cmake-3.30.1-windows-i386/bin/cmake.exe" -DCURRENT_PORT_DIR=C:/Users/Yeet/Tauon/vcpkg/scripts/detect_compiler -DCURRENT_BUILDTREES_DIR=C:/Users/Yeet/Tauon/vcpkg/buildtrees/detect_compiler -DCURRENT_PACKAGES_DIR=C:/Users/Yeet/Tauon/vcpkg/packages/detect_compiler_x64-windows -D_HOST_TRIPLET=x64-windows -DCMD=BUILD -DDOWNLOADS=C:/Users/Yeet/Tauon/vcpkg/downloads -DTARGET_TRIPLET=x64-windows -DTARGET_TRIPLET_FILE=C:/Users/Yeet/Tauon/vcpkg/triplets/x64-windows.cmake -DVCPKG_BASE_VERSION=2024-11-12 -DVCPKG_CONCURRENCY=3 -DVCPKG_PLATFORM_TOOLSET=v143 "-DGIT=C:/Program Files/Git/cmd/git.exe" -DVCPKG_ROOT_DIR=C:/Users/Yeet/Tauon/vcpkg -DPACKAGES_DIR=C:/Users/Yeet/Tauon/vcpkg/packages -DBUILDTREES_DIR=C:/Users/Yeet/Tauon/vcpkg/buildtrees -D_VCPKG_INSTALLED_DIR=C:/Users/Yeet/Tauon/vcpkg/installed -DDOWNLOADS=C:/Users/Yeet/Tauon/vcpkg/downloads -DVCPKG_MANIFEST_INSTALL=OFF -P "C:\Users\Yeet\Tauon\vcpkg\scripts\ports.cmake")
[DEBUG] -- Found external ninja('1.12.1').
[DEBUG] -- Configuring x64-windows
[DEBUG] [1/1] "C:/Users/Yeet/Tauon/vcpkg/downloads/tools/cmake-3.30.1-windows/cmake-3.30.1-windows-i386/bin/cmake.exe" -E chdir ".." "C:/Users/Yeet/Tauon/vcpkg/downloads/tools/cmake-3.30.1-windows/cmake-3.30.1-windows-i386/bin/cmake.exe" "C:/Users/Yeet/Tauon/vcpkg/scripts/detect_compiler" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Release" "-DCMAKE_INSTALL_PREFIX=C:/Users/Yeet/Tauon/vcpkg/packages/detect_compiler_x64-windows" "-DCMAKE_MAKE_PROGRAM=C:/Program Files (x86)/Microsoft Visual Studio/2022/BuildTools/Common7/IDE/CommonExtensions/Microsoft/CMake/Ninja/ninja.exe" "-DBUILD_SHARED_LIBS=ON" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=C:/Users/Yeet/Tauon/vcpkg/scripts/toolchains/windows.cmake" "-DVCPKG_TARGET_TRIPLET=x64-windows" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=v143" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=C:/Users/Yeet/Tauon/vcpkg/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=C:/Users/Yeet/Tauon/vcpkg" "-DZ_VCPKG_ROOT_DIR=C:/Users/Yeet/Tauon/vcpkg" "-D_VCPKG_INSTALLED_DIR=C:/Users/Yeet/Tauon/vcpkg/installed" "-DVCPKG_MANIFEST_INSTALL=OFF"
[DEBUG] -- The C compiler identification is MSVC 19.42.34433.0
[DEBUG] -- The CXX compiler identification is MSVC 19.42.34433.0
[DEBUG] #COMPILER_HASH#375a3995ef48384963d6e03880af65bc52efbc2a
[DEBUG] #COMPILER_C_HASH#c1103da064e69b2cfbd07929fc2ede52b5895dd7
[DEBUG] #COMPILER_C_VERSION#19.42.34433.0
[DEBUG] #COMPILER_C_ID#MSVC
[DEBUG] #COMPILER_C_PATH#C:/Program Files (x86)/Microsoft Visual Studio/2022/BuildTools/VC/Tools/MSVC/14.42.34433/bin/Hostx64/x64/cl.exe
[DEBUG] #COMPILER_CXX_HASH#c1103da064e69b2cfbd07929fc2ede52b5895dd7
[DEBUG] #COMPILER_CXX_VERSION#19.42.34433.0
[DEBUG] #COMPILER_CXX_ID#MSVC
[DEBUG] #COMPILER_CXX_PATH#C:/Program Files (x86)/Microsoft Visual Studio/2022/BuildTools/VC/Tools/MSVC/14.42.34433/bin/Hostx64/x64/cl.exe
[DEBUG] -- Configuring done (1.2s)
[DEBUG] -- Generating done (0.0s)
[DEBUG] CMake Warning:
[DEBUG]   Manually-specified variables were not used by the project:
[DEBUG]
[DEBUG]     BUILD_SHARED_LIBS
[DEBUG]     CMAKE_INSTALL_BINDIR
[DEBUG]     CMAKE_INSTALL_LIBDIR
[DEBUG]     _VCPKG_ROOT_DIR
[DEBUG]
[DEBUG]
[DEBUG] -- Build files have been written to: C:/Users/Yeet/Tauon/vcpkg/buildtrees/detect_compiler/x64-windows-rel
[DEBUG]
[DEBUG] 1008: cmd_execute_and_stream_data() returned 0 after  1983195 us
[DEBUG] Detected compiler hash for triplet x64-windows: 375a3995ef48384963d6e03880af65bc52efbc2a
Compiler found: C:/Program Files (x86)/Microsoft Visual Studio/2022/BuildTools/VC/Tools/MSVC/14.42.34433/bin/Hostx64/x64/cl.exe
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\ports\mpg123\have-fpu.diff
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\ports\mpg123\have-fpu.diff has hash 9eb6c504ccb0577854a7af6b64efbb9575631bbd589afad5a4aa6f014634212b
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\ports\mpg123\vcpkg.json
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\ports\mpg123\vcpkg.json has hash c0c5834741d2dc0986f6fe5f14b0483c3e0749002c61e912acae14336f9c3f72
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\compile_wrapper_consider_clang-cl.patch
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\compile_wrapper_consider_clang-cl.patch has hash 12dbcf92febc9400d782a3d7d073acee4b09a1417ce7d4d5e9be7c55aa177d36
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\execute_process.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\execute_process.cmake has hash 66a937b9c074422643135c319d1abadaa45484a664f1b160d4c163efb444a446
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_acquire_msys.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_acquire_msys.cmake has hash 251a30badbb2082236811ba6975680d0d012a20dd55db7e39ccbc6cc49d67287
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_add_to_path.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_add_to_path.cmake has hash 5f5ae75cf37b2a58d1a8561ca96496b64cd91ec9a0afab0b976c3e5d59030bfe
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_apply_patches.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_apply_patches.cmake has hash 4c123ce6f9bf0c46adc8287b7bd6a40b5f5e8daa83623dd21bb8c568ad4421fc
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_backup_restore_env_vars.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_backup_restore_env_vars.cmake has hash 5621f3b6a9586ffc02d49a5032560a48ad291de1cce553b904588f0108648904
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_buildpath_length_warning.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_buildpath_length_warning.cmake has hash fa440f3734e34b462ef96f3e89f0a9c5a11b609110a15a162d24da9218693fd1
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_build_cmake.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_build_cmake.cmake has hash 5201d6149d85fff414f3fbbdc4c793e0b2aeeb6d249a5ae8cf15e39fef66b7e6
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_build_make.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_build_make.cmake has hash e6ad744274e4e686395953a69e41afbf0d01f532aacf3ad7c5b444fb1ec14621
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_build_msbuild.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_build_msbuild.cmake has hash 9a8367ac19733fc51b8e7edc77968725d2efe1400aaf1fdd767bb329a7b04532
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_build_ninja.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_build_ninja.cmake has hash 9bafdd942fe41be20a868e0a41ea6f983895bacba2f257921b53d789c7ad8092
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_build_nmake.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_build_nmake.cmake has hash 04f244a60749279ee930d3ad9d6f1a659ed280fbd958b3d84c73b10ade4e501c
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_build_qmake.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_build_qmake.cmake has hash cd00146eb665b004c527cf74986e392d2419610ace073bd6abb79d55a7912004
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_check_features.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_check_features.cmake has hash 943b217e0968d64cf2cb9c272608e6a0b497377e792034f819809a79e1502c2b
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_check_linkage.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_check_linkage.cmake has hash 2236edfea13af20524c17d73214e0e5111bf1c13a2fc46c7f2912a46abd0c1a8
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_clean_executables_in_bin.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_clean_executables_in_bin.cmake has hash 5f7427ddf120583cf1f429861989e937858ee7a8c4deb419dca6f3f2fee86a65
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_clean_msbuild.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_clean_msbuild.cmake has hash 67007e54dee8a38487b43228573d877ad270fc6bac7bf8849783d0c2a78ce501
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_common_definitions.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_common_definitions.cmake has hash 3ff4e55e804dbd45bd66ed4af378df9c6364e6ce08bc741dfff3b99bcd8653dc
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_common_functions.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_common_functions.cmake has hash 5f07e6f6184a02dcd8fc058eed1747a6bb65ee4a8f524e717651674f0bd7904c
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_configure_cmake.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_configure_cmake.cmake has hash 9dfd362bd20613eaa83af55eb0f98c8cb50fd4826a65da74a0f1641da73497c2
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_configure_gn.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_configure_gn.cmake has hash c6637dbdb7c20b2e067c924eeb50605f7e3cc9e7c8c1434e53c9280134df11ba
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_configure_make.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_configure_make.cmake has hash 1e2d99e6035a77741312f71dc56f8b339f63b03fe5bea18971a052f624ef49a9
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_configure_meson.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_configure_meson.cmake has hash ee93cbbf7c8d52f4a401a158dfc0613ef9516ddb4dd03bc944d2d1351288961c
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_configure_qmake.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_configure_qmake.cmake has hash 5569fcdfd8bb3ff582baf4f080f565c2d6c048872af5a4987818d19eaafa0663
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_copy_pdbs.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_copy_pdbs.cmake has hash 921d35f6fd2d29a55e55856f87c2da06eeb6c7f0a3c5cedfaf0d9708f061ad62
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_copy_tools.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_copy_tools.cmake has hash 3d45ff761bddbabe8923b52330168dc3abd295fa469d3f2e47cb14dce85332d5
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_copy_tool_dependencies.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_copy_tool_dependencies.cmake has hash 66eef4116a52fd8cba76d274f44904846df943a3fc52317871cf255733b2efc4
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_download_distfile.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_download_distfile.cmake has hash 3aa6247848f15f0b54032041e0dc6bbdebbb3f3b1c24a42d5fa998efd2d74753
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_download_sourceforge.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_download_sourceforge.cmake has hash d8a6ab3f2cd82c41cc9a9b9ea9a75f84d7b3a92ac7b3c5c9bb95d381bbae1233
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_execute_build_process.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_execute_build_process.cmake has hash 4976d00fc7d25ad07984f282490121a09aa44a49c5dae627ca68355affd929d0
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_execute_in_download_mode.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_execute_in_download_mode.cmake has hash 75a3cabe04d6bbaf1f73555516dd3c492edbed22cea204835c3620db5d948851
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_execute_required_process.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_execute_required_process.cmake has hash 2df167e6e7f37c8038e02a3ebbfb239fa55de0b384ed4532d33f5483710a608b
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_execute_required_process_repeat.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_execute_required_process_repeat.cmake has hash 4863f71de8d0fd53ed799c6a5d15795156ae0606ecb61489b7636a050e029e2b
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_extract_archive.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_extract_archive.cmake has hash f115b4f0fee25842e4eb32f6dd130aeee2cfd8c049606bcf86fcf6f509076014
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_extract_source_archive.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_extract_source_archive.cmake has hash 9c733832f8b58b05e6c329ae8b6ad1cfa858224fdc06d4a2d08b428e31de1a51
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_extract_source_archive_ex.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_extract_source_archive_ex.cmake has hash ab9f5bd71a8e1ca4a56d5c09f248c2bc10ff8df3b600c3efb6cca34cba9ec3b7
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_fail_port_install.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_fail_port_install.cmake has hash aef80df96f7c83d79e7b03d132161d8071228e0f1c9eb61ba8b16354f0c9496c
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_find_acquire_program(7Z).cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_find_acquire_program(7Z).cmake has hash 67e3935c9b8165f2b91b34b48fdb7bcc1e2a6d20cf2b0a4e63b9bb787c9dad1e
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_find_acquire_program(ARIA2).cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_find_acquire_program(ARIA2).cmake has hash 948da78edc0b50c8ab8dd8acee7687d3fbbdc6f7c1df2ee8e2bf5fe673794fbe
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_find_acquire_program(BAZEL).cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_find_acquire_program(BAZEL).cmake has hash 16409b8196d7bb13a7007ecaaaff81d204a8b1194747590136cb517c50547dc5
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_find_acquire_program(BISON).cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_find_acquire_program(BISON).cmake has hash 97a601e71044956e42e0e51183df1bcd51e9a9490fc3c2e79da0368889b28b64
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_find_acquire_program(CLANG).cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_find_acquire_program(CLANG).cmake has hash bc9f8433d5ce5a51e45a79c7469cdd9fcad154488b33ff6353af3b29400af50d
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_find_acquire_program(DARK).cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_find_acquire_program(DARK).cmake has hash 65a32f6f8a7707456a9e96143ea1e438fb44c82ebddf881ff0efc470754577b0
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_find_acquire_program(DOXYGEN).cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_find_acquire_program(DOXYGEN).cmake has hash 8f4ac6c2f60e1e89e5b48a241098a63141e0d88f00d5fdc2d3c2846c9eaeb82b
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_find_acquire_program(FLEX).cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_find_acquire_program(FLEX).cmake has hash 5126699579bda4fb2b605adedd8cfed7ac53430178931ffd20d8b007cc89f26f
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_find_acquire_program(GASPREPROCESSOR).cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_find_acquire_program(GASPREPROCESSOR).cmake has hash 0ff31ab283d7b78bc7af20cdbd66b5adc0432f00f126f8111003245908398c88
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_find_acquire_program(GIT).cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_find_acquire_program(GIT).cmake has hash bb3644b8863c1a03c3c76cf8b13047557e05b22f4be7a13cb4ad00e72d7ad6b6
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_find_acquire_program(GN).cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_find_acquire_program(GN).cmake has hash 2df6666d24cc24c550f96d4fee8140656ff6739fdbc57c4a774f56c7d85ff2fa
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_find_acquire_program(GO).cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_find_acquire_program(GO).cmake has hash 47999a9e8fc92fb7b45fcb1ac1f22d4b69a0b3fff83da2e8df8850fd824ad03f
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_find_acquire_program(GPERF).cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_find_acquire_program(GPERF).cmake has hash 07129ac2ca1559ec4dc10e5e24efa25303e380e801049ef548b278a6944cd036
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_find_acquire_program(JOM).cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_find_acquire_program(JOM).cmake has hash a3409f6506dcb4694592483964e680298244e26233900c6b6d95a9e18866b580
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_find_acquire_program(MESON).cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_find_acquire_program(MESON).cmake has hash b39aa03a7f091406f39101eadd5ede4f893f01cc2a825422be9b7ad537e64a69
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_find_acquire_program(NASM).cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_find_acquire_program(NASM).cmake has hash b6fcd010dcc70cec5e2472dde5a5b794c11ed63a433215d0e4f8ab11620cde0a
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_find_acquire_program(NINJA).cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_find_acquire_program(NINJA).cmake has hash 8692aff2f893a8afcabbb4395a8b91ad4f6dd228f410fa4fd050d99c2ec0f52c
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_find_acquire_program(NUGET).cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_find_acquire_program(NUGET).cmake has hash 1123c7214f2942acaff8e024c492227d6ef264abc7e4e815517931cbf3796d1d
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_find_acquire_program(PATCHELF).cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_find_acquire_program(PATCHELF).cmake has hash 5a09332e14938aec2529bbabd0e174a33549032dc699ede8ea66bdcc4fe1a9af
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_find_acquire_program(PERL).cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_find_acquire_program(PERL).cmake has hash 187adedd1721a961e70595c183a93ab5ead976e340be77690c551f4c7f0111dd
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_find_acquire_program(PKGCONFIG).cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_find_acquire_program(PKGCONFIG).cmake has hash 12bca5c33b8bb8e2edf92ffc410313fd7b7dbe2433e61d018fe51df2599ea7e3
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_find_acquire_program(PYTHON2).cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_find_acquire_program(PYTHON2).cmake has hash ed078b84af22461e59694ab205a5ba9d0eb77d094ea64fc5754f7e19644b052e
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_find_acquire_program(PYTHON3).cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_find_acquire_program(PYTHON3).cmake has hash 3f22a4ae610d824ca9381116fa761819c18510ed754d7676c4560bade78964d7
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_find_acquire_program(RUBY).cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_find_acquire_program(RUBY).cmake has hash 641f9c542a6b65ecdf7626d4a9dadee1c1ee6e8b92bbf8ec184969a514fbf28e
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_find_acquire_program(SCONS).cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_find_acquire_program(SCONS).cmake has hash 98cfd79a1230600d571576286abef2bc2b145ee16892d92f7b2cc60c3589dd87
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_find_acquire_program(SWIG).cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_find_acquire_program(SWIG).cmake has hash 4b07ff329c995f4ecb6b4c325b690757f2b4ec6666ee7b2264d7083e218fd35c
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_find_acquire_program(YASM).cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_find_acquire_program(YASM).cmake has hash a0e60a6625d0b66f43405d8c438c6241154037c16358fbb14e66f834703710df
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_find_acquire_program.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_find_acquire_program.cmake has hash 03722d714d388b19731f5c0be35996c25a47d5b35dd6a09669ea1e599a56e005
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_find_fortran.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_find_fortran.cmake has hash 95e1024549080e904e8a499c458029bb27dcc3278ab404293a844e75b836a026
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_fixup_cmake_targets.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_fixup_cmake_targets.cmake has hash 7f777f83ca76540ed573d986ee53d1b3da1e45f49f3856c44c0773fec71cb49f
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_fixup_pkgconfig.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_fixup_pkgconfig.cmake has hash 1a15f6c6d8e2b244d83a7514a0412d339127d2217d1df60ad1388b546c85f777
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_from_bitbucket.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_from_bitbucket.cmake has hash e875eb81fb8b06b08d0c97aed211b161dca3d1b7da8e562aa5ee8a8809e9e3d0
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_from_git.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_from_git.cmake has hash 96ed81968f76354c00096dd8cd4e63c6a235fa969334a11ab18d11c0c512ff58
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_from_github.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_from_github.cmake has hash b743742296a114ea1b18ae99672e02f142c4eb2bef7f57d36c038bedbfb0502f
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_from_gitlab.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_from_gitlab.cmake has hash 5bbb6c56a4be11c0d10d488d95af91a37b15f520eba862254a24c6665dd39588
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_from_sourceforge.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_from_sourceforge.cmake has hash 00cb7d5767d56fdd8a1715ebd3c159fccd44dc17653522e23d2e507bce44a4f8
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_get_program_files_platform_bitness.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_get_program_files_platform_bitness.cmake has hash 0c21ce7e9fcf7828f2e06f488f4675816b87942cfbf828c183f5bbcba8ae5fc7
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_get_windows_sdk.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_get_windows_sdk.cmake has hash 1bb57ecf48db9703b9496a40ff9eeec83b856362b426d23e7d25a54a0ac55916
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_host_path_list.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_host_path_list.cmake has hash 1e8382b3a80fc7458293fce1659077dcacda9892eb63d911ae0330fdb1aedc2d
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_install_cmake.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_install_cmake.cmake has hash 6119891902c6ad9e9ed1d9b073eb9969159b553f9d6f5bee529cde91769355ff
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_install_copyright.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_install_copyright.cmake has hash ba6c169ab4e59fa05682e530cdeb883767de22c8391f023d4e6844a7ec5dd3d2
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_install_gn.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_install_gn.cmake has hash 18f1e80697fca670452e1ff181850e546764cc7c5fa8cb5d6442266523af9806
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_install_make.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_install_make.cmake has hash 8d8145837ce718ef60778b50fea1af272146c9efa42367ec2d7726332ef29ab5
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_install_meson.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_install_meson.cmake has hash cc77e0248f5e935134c9061e5d424bc6533b561eeeac6f72d1788837cd5d0384
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_install_msbuild.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_install_msbuild.cmake has hash d770323312cc635d641183bfa4a0c073f919c8a37b7d9b3576700b1e934f8cd4
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_install_nmake.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_install_nmake.cmake has hash e5540f04f33bc8eee3417da64e2c9a9aa2bc3dee62031238d7eb795ec31cb0b7
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_install_qmake.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_install_qmake.cmake has hash e9eedf4e8a942dbae43e25d0f7a38a8c3afe7761c5d9e042d1a52017e3dcebe6
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_list.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_list.cmake has hash f5de3ebcbc40a4db90622ade9aca918e2cf404dc0d91342fcde457d730e6fa29
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_minimum_required.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_minimum_required.cmake has hash 4da8ab803b83c8f1c7b6272298108b9f62a7cd0618e563242c8e584692534171
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_replace_string.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_replace_string.cmake has hash b450deb79207478b37119743e00808ebc42de0628e7b98c14ab24728bd5c78b8
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_test_cmake.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\vcpkg_test_cmake.cmake has hash 8117a509de7f30dd5c1f4c35134c1b3180c0825ea03480aa4625e98ddd5476e6
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\z_vcpkg_apply_patches.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\z_vcpkg_apply_patches.cmake has hash aed81b9cb2c44e2736e6e912445044a2f0512c041ecbb72ffb3154e44b0044ba
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\z_vcpkg_escape_regex_control_characters.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\z_vcpkg_escape_regex_control_characters.cmake has hash 439c52c6ea37cecad4474528fb3b8019f2538380b3771d986e3d7811d8e274b5
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\z_vcpkg_fixup_rpath.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\z_vcpkg_fixup_rpath.cmake has hash fbe6e9f100b14ca5aa62df2d7fb3dfa5835a7f4eff4690dca0421985ca0a7780
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\z_vcpkg_fixup_rpath_macho.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\z_vcpkg_fixup_rpath_macho.cmake has hash 8ef303331d7e75d45035d1c00b9f2703380ed3a629084158a3c5e1dc26f9c6ac
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\z_vcpkg_forward_output_variable.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\z_vcpkg_forward_output_variable.cmake has hash ae3440b77321516f29f0b51e26b8ecb05fd31f63d33949dc406a502c300b04ff
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\z_vcpkg_function_arguments.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\z_vcpkg_function_arguments.cmake has hash 11721a8a42a2aa2f6f285d485b21aed55834c2029140661bd8e2f0b709288d77
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\z_vcpkg_get_cmake_vars.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\z_vcpkg_get_cmake_vars.cmake has hash ce74da978727f15cc4eab6fa8a9c27023f0b51ef654cfbf7df8f3409fb950c43
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\z_vcpkg_prettify_command_line.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\z_vcpkg_prettify_command_line.cmake has hash 97e4e92ac6e9da544fc7b9a6695e0d491c2c095ec265d8c0a90bcb8168873ca3
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\z_vcpkg_setup_pkgconfig_path.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\cmake\z_vcpkg_setup_pkgconfig_path.cmake has hash ed25792f3b2d24819e143623be1f846513f5955cf7808de0a88b0ecaf97d4ea7
[DEBUG] Trying to hash C:\Users\Yeet\Tauon\vcpkg\scripts\ports.cmake
[DEBUG] C:\Users\Yeet\Tauon\vcpkg\scripts\ports.cmake has hash 02fb6521157895d67c3b6e17068a30d814bafacd0aaa41b43c9f1ea57b795ace
[DEBUG] <abientries for mpg123:x64-windows>
[DEBUG]   cmake|3.30.1
[DEBUG]   features|core
[DEBUG]   have-fpu.diff|9eb6c504ccb0577854a7af6b64efbb9575631bbd589afad5a4aa6f014634212b
[DEBUG]   portfile.cmake|d8ee4ec115f9796d100797c39cb64a2a03adc4eb2066b7c108c46d98cd68de3a
[DEBUG]   ports.cmake|02fb6521157895d67c3b6e17068a30d814bafacd0aaa41b43c9f1ea57b795ace
[DEBUG]   post_build_checks|2
[DEBUG]   powershell|7.4.6
[DEBUG]   triplet|x64-windows
[DEBUG]   triplet_abi|4556164a2cd3dd6f4742101eabb46def7e71b6e5856faa88e5d005aac12a803c-0433e2303d8e9448dd9cf29d3052811cb89f6324bfd45703cfd4ec1e4134228d-375a3995ef48384963d6e03880af65bc52efbc2a
[DEBUG]   vcpkg-cmake|15759b1ecff16f5de1c1314f961260b7db0e37bf135d43e2f568433e4184d13c
[DEBUG]   vcpkg-cmake-config|c87fd0db179d9c64cd4b84a365647442714ac1a9eef5545229b653bf1950a4b1
[DEBUG]   vcpkg.json|c0c5834741d2dc0986f6fe5f14b0483c3e0749002c61e912acae14336f9c3f72
[DEBUG]   vcpkg_copy_pdbs|921d35f6fd2d29a55e55856f87c2da06eeb6c7f0a3c5cedfaf0d9708f061ad62
[DEBUG]   vcpkg_fixup_pkgconfig|1a15f6c6d8e2b244d83a7514a0412d339127d2217d1df60ad1388b546c85f777
[DEBUG]   vcpkg_from_sourceforge|00cb7d5767d56fdd8a1715ebd3c159fccd44dc17653522e23d2e507bce44a4f8
[DEBUG]   vcpkg_install_copyright|ba6c169ab4e59fa05682e530cdeb883767de22c8391f023d4e6844a7ec5dd3d2
[DEBUG]   vcpkg_list|f5de3ebcbc40a4db90622ade9aca918e2cf404dc0d91342fcde457d730e6fa29
[DEBUG]   yasm|b49c179a5ccbf5a8e6bb36f1d57a6ba0fd76c210b1b9cbc9a22499e660340ec6
[DEBUG] </abientries>
[DEBUG] Default binary cache path is: C:\Users\Yeet\AppData\Local\vcpkg\archives
Restored 0 package(s) from C:\Users\Yeet\AppData\Local\vcpkg\archives in 88.4 us. Use --debug to see more details.
Installing 1/1 mpg123:x64-windows@1.32.9...
Building mpg123:x64-windows@1.32.9...
[DEBUG] Detecting host with IsWow64Process2
[DEBUG] 1009: CreateProcessW("C:\Users\Yeet\Tauon\vcpkg\downloads\tools\cmake-3.30.1-windows\cmake-3.30.1-windows-i386/bin/cmake.exe" -DALL_FEATURES= -DCURRENT_PORT_DIR=C:/Users/Yeet/Tauon/vcpkg/ports/mpg123 -D_HOST_TRIPLET=x64-windows -DFEATURES=core -DPORT=mpg123 -DVERSION=1.32.9 -DVCPKG_USE_HEAD_VERSION=0 -D_VCPKG_DOWNLOAD_TOOL=BUILT_IN -D_VCPKG_EDITABLE=0 -D_VCPKG_NO_DOWNLOADS=0 -DZ_VCPKG_CHAINLOAD_TOOLCHAIN_FILE=C:/Users/Yeet/Tauon/vcpkg/scripts/toolchains/windows.cmake -DZ_VCPKG_POST_PORTFILE_INCLUDES= -DCMD=BUILD -DDOWNLOADS=C:/Users/Yeet/Tauon/vcpkg/downloads -DTARGET_TRIPLET=x64-windows -DTARGET_TRIPLET_FILE=C:/Users/Yeet/Tauon/vcpkg/triplets/x64-windows.cmake -DVCPKG_BASE_VERSION=2024-11-12 -DVCPKG_CONCURRENCY=3 -DVCPKG_PLATFORM_TOOLSET=v143 "-DGIT=C:/Program Files/Git/cmd/git.exe" "-DVCPKG_PORT_CONFIGS=C:\Users\Yeet\Tauon\vcpkg\installed\x64-windows\share\vcpkg-cmake\vcpkg-port-config.cmake;C:\Users\Yeet\Tauon\vcpkg\installed\x64-windows\share\vcpkg-cmake-config\vcpkg-port-config.cmake;C:\Users\Yeet\Tauon\vcpkg\installed\x64-windows\share\yasm\vcpkg-port-config.cmake" -DVCPKG_ROOT_DIR=C:/Users/Yeet/Tauon/vcpkg -DPACKAGES_DIR=C:/Users/Yeet/Tauon/vcpkg/packages -DBUILDTREES_DIR=C:/Users/Yeet/Tauon/vcpkg/buildtrees -D_VCPKG_INSTALLED_DIR=C:/Users/Yeet/Tauon/vcpkg/installed -DDOWNLOADS=C:/Users/Yeet/Tauon/vcpkg/downloads -DVCPKG_MANIFEST_INSTALL=OFF -P "C:\Users\Yeet\Tauon\vcpkg\scripts\ports.cmake")
-- Using cached mpg123-1.32.9.tar.bz2.
-- Extracting source C:/Users/Yeet/Tauon/vcpkg/downloads/mpg123-1.32.9.tar.bz2
-- Applying patch have-fpu.diff
-- Using source at C:/Users/Yeet/Tauon/vcpkg/buildtrees/mpg123/src/mpg123-1-0e4801c005.clean
-- Found external ninja('1.12.1').
-- Configuring x64-windows
-- Building x64-windows-dbg
-- Building x64-windows-rel
-- Fixing pkgconfig file: C:/Users/Yeet/Tauon/vcpkg/packages/mpg123_x64-windows/lib/pkgconfig/libmpg123.pc
-- Fixing pkgconfig file: C:/Users/Yeet/Tauon/vcpkg/packages/mpg123_x64-windows/lib/pkgconfig/libout123.pc
-- Fixing pkgconfig file: C:/Users/Yeet/Tauon/vcpkg/packages/mpg123_x64-windows/lib/pkgconfig/libsyn123.pc
-- Using cached msys2-mingw-w64-x86_64-pkgconf-1~2.3.0-1-any.pkg.tar.zst.
-- Using cached msys2-msys2-runtime-3.5.4-2-x86_64.pkg.tar.zst.
-- Using msys root at C:/Users/Yeet/Tauon/vcpkg/downloads/tools/msys2/21caed2f81ec917b
-- Fixing pkgconfig file: C:/Users/Yeet/Tauon/vcpkg/packages/mpg123_x64-windows/debug/lib/pkgconfig/libmpg123.pc
-- Fixing pkgconfig file: C:/Users/Yeet/Tauon/vcpkg/packages/mpg123_x64-windows/debug/lib/pkgconfig/libout123.pc
-- Fixing pkgconfig file: C:/Users/Yeet/Tauon/vcpkg/packages/mpg123_x64-windows/debug/lib/pkgconfig/libsyn123.pc
-- Installing: C:/Users/Yeet/Tauon/vcpkg/packages/mpg123_x64-windows/share/mpg123/copyright
[DEBUG] 1009: cmd_execute_and_stream_data() returned 0 after 53986380 us
-- Performing post-build validation
[DEBUG] The lib C:\Users\Yeet\Tauon\vcpkg\packages\mpg123_x64-windows\debug\lib\mpg123.lib has directives:
[DEBUG] The lib C:\Users\Yeet\Tauon\vcpkg\packages\mpg123_x64-windows\debug\lib\out123.lib has directives:
[DEBUG] The lib C:\Users\Yeet\Tauon\vcpkg\packages\mpg123_x64-windows\debug\lib\syn123.lib has directives:
[DEBUG] The lib C:\Users\Yeet\Tauon\vcpkg\packages\mpg123_x64-windows\lib\mpg123.lib has directives:
[DEBUG] The lib C:\Users\Yeet\Tauon\vcpkg\packages\mpg123_x64-windows\lib\out123.lib has directives:
[DEBUG] The lib C:\Users\Yeet\Tauon\vcpkg\packages\mpg123_x64-windows\lib\syn123.lib has directives:
[DEBUG] 1010: CreateProcessW("C:\Users\Yeet\Tauon\vcpkg\downloads\tools\7zip-24.08-windows\7za.exe" a "
C:\Users\Yeet\Tauon\vcpkg\packages\mpg123_x64-windows.zip" "C:\Users\Yeet\Tauon\vcpkg\packages\mpg123_x64-windows\*")
[DEBUG] 1010: cmd_execute_and_stream_data() returned 0 after   660320 us
Stored binaries in 1 destinations in 663 ms.
Elapsed time to handle mpg123:x64-windows: 55 s
mpg123:x64-windows package ABI: 5d9a23259decc40d416ea51d92c06ec670b966d77072bb67ef409d5ac34ce63d
Total install time: 55 s
mpg123 provides CMake targets:

  # this is heuristically generated, and may not be correct
  find_package(mpg123 CONFIG REQUIRED)
  target_link_libraries(main PRIVATE MPG123::libmpg123 MPG123::libout123 MPG123::libsyn123)

mpg123 provides pkg-config modules:

  # An optimised MPEG Audio decoder
  libmpg123

  # A streaming audio output API derived from mpg123
  libout123

  # A signal synthesis library accompanying mpg123
  libsyn123

[DEBUG] D:\a\_work\1\s\src\vcpkg\commands.install.cpp(1411):
[DEBUG] Time in subprocesses: 59613026us
[DEBUG] Time in parsing JSON: 320us
[DEBUG] Time in JSON reader: 475us
[DEBUG] Time in filesystem: 256916us
[DEBUG] Time in loading ports: 128760us
[DEBUG] Exiting after 1 min (60616533us)
```

```powershell
PS C:\Users\Yeet\Tauon\vcpkg> &"C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\14.42.34433\bin\Hostx64\x64\dumpbin.exe" /EXPORTS C:\Users\Yeet\Tauon\vcpkg\installed\x64-windows\lib\mpg123.lib
Microsoft (R) COFF/PE Dumper Version 14.42.34433.0
Copyright (C) Microsoft Corporation.  All rights reserved.


Dump of file C:\Users\Yeet\Tauon\vcpkg\installed\x64-windows\lib\mpg123.lib

File Type: LIBRARY

     Exports

       ordinal    name

                  mpg123_add_string
                  mpg123_add_substring
                  mpg123_chomp_string
                  mpg123_clip
                  mpg123_close
                  mpg123_copy_string
                  mpg123_current_decoder
                  mpg123_decode
                  mpg123_decode_frame
                  mpg123_decode_frame64
                  mpg123_decoder
                  mpg123_decoders
                  mpg123_delete
                  mpg123_delete_pars
                  mpg123_delete_string
                  mpg123_enc_from_id3
                  mpg123_enc_from_id3_2
                  mpg123_encodings
                  mpg123_encsize
                  mpg123_eq
                  mpg123_eq2
                  mpg123_eq_bands
                  mpg123_eq_change
                  mpg123_errcode
                  mpg123_exit
                  mpg123_feature
                  mpg123_feature2
                  mpg123_feed
                  mpg123_feedseek
                  mpg123_feedseek64
                  mpg123_fmt
                  mpg123_fmt2
                  mpg123_fmt_all
                  mpg123_fmt_none
                  mpg123_fmt_support
                  mpg123_format
                  mpg123_format2
                  mpg123_format_all
                  mpg123_format_none
                  mpg123_format_support
                  mpg123_framebyframe_decode
                  mpg123_framebyframe_decode64
                  mpg123_framebyframe_next
                  mpg123_framedata
                  mpg123_framelength
                  mpg123_framelength64
                  mpg123_framepos
                  mpg123_framepos64
                  mpg123_free
                  mpg123_free_string
                  mpg123_geteq
                  mpg123_geteq2
                  mpg123_getformat
                  mpg123_getformat2
                  mpg123_getpar
                  mpg123_getpar2
                  mpg123_getparam
                  mpg123_getparam2
                  mpg123_getstate
                  mpg123_getstate2
                  mpg123_getvolume
                  mpg123_grow_string
                  mpg123_icy
                  mpg123_icy2utf8
                  mpg123_id3
                  mpg123_id3_raw
                  mpg123_index
                  mpg123_index64
                  mpg123_info
                  mpg123_info2
                  mpg123_init
                  mpg123_init_string
                  mpg123_length
                  mpg123_length64
                  mpg123_meta_check
                  mpg123_meta_free
                  mpg123_move_string
                  mpg123_new
                  mpg123_new_pars
                  mpg123_new_string
                  mpg123_open_feed
                  mpg123_open_handle
                  mpg123_outblock
                  mpg123_par
                  mpg123_par2
                  mpg123_param
                  mpg123_param2
                  mpg123_parnew
                  mpg123_plain_strerror
                  mpg123_position
                  mpg123_rates
                  mpg123_read
                  mpg123_reader64
                  mpg123_replace_buffer
                  mpg123_replace_reader
                  mpg123_replace_reader_handle
                  mpg123_reset_eq
                  mpg123_resize_string
                  mpg123_safe_buffer
                  mpg123_same_string
                  mpg123_scan
                  mpg123_seek
                  mpg123_seek64
                  mpg123_seek_frame
                  mpg123_seek_frame64
                  mpg123_set_filesize
                  mpg123_set_filesize64
                  mpg123_set_index
                  mpg123_set_index64
                  mpg123_set_moreinfo
                  mpg123_set_string
                  mpg123_set_substring
                  mpg123_spf
                  mpg123_store_utf8
                  mpg123_store_utf8_2
                  mpg123_strerror
                  mpg123_strlen
                  mpg123_supported_decoders
                  mpg123_tell
                  mpg123_tell64
                  mpg123_tell_stream
                  mpg123_tell_stream64
                  mpg123_tellframe
                  mpg123_tellframe64
                  mpg123_timeframe
                  mpg123_timeframe64
                  mpg123_tpf
                  mpg123_volume
                  mpg123_volume_change
                  mpg123_volume_change_db

  Summary

          C0 .debug$S
          14 .idata$2
          14 .idata$3
           8 .idata$4
           8 .idata$5
           C .idata$6
```

</details>

--- a/ports/mpg123/fix-dllexport.diff
+++ b/ports/mpg123/fix-dllexport.diff
@@ -1,0 +1,16 @@
+diff --git a/src/libmpg123/mpg123lib_intern.h b/src/libmpg123/mpg123lib_intern.h
+index b46c419..7cd5504 100644
+--- a/src/libmpg123/mpg123lib_intern.h
++++ b/src/libmpg123/mpg123lib_intern.h
+@@ -21,7 +21,10 @@
+ 
+ // Only portable API plays a role in the library itself, outside of lfs_wrap.c.
+ // Also, we need to ensure no suffix renaming for the primary implementations.
+-#define MPG123_PORTABLE_API
++// But: The _definition_ of non-portable API needs to be present for those
++// primary implementations being exported for DLL builds. Just the largefile
++// renaming needs to be skipped!
++#define MPG123_NO_LARGENAME
+ #define MPG123_ENUM_API
+ #include "mpg123.h"
+ 

--- a/ports/mpg123/pkgconfig.diff
+++ b/ports/mpg123/pkgconfig.diff
@@ -1,0 +1,66 @@
+diff --git a/ports/cmake/src/CMakeLists.txt b/ports/cmake/src/CMakeLists.txt
+index 32bc03c..952141b 100644
+--- a/ports/cmake/src/CMakeLists.txt
++++ b/ports/cmake/src/CMakeLists.txt
+@@ -450,3 +450,7 @@ if(UNIX AND NOT PORTABLE_API)
+ endif()
+ 
+ configure_file(config.cmake.h.in config.h)
++
++set(LIBMPG123_LIBS "${LIBMPG123_LIBS}" PARENT_SCOPE)
++set(LIBOUT123_LIBS "${LIBOUT123_LIBS}" PARENT_SCOPE)
++set(LIBSYN123_LIBS "${LIBSYN123_LIBS}" PARENT_SCOPE)
+diff --git a/ports/cmake/src/libmpg123/CMakeLists.txt b/ports/cmake/src/libmpg123/CMakeLists.txt
+index 7cb08be..3e1c042 100644
+--- a/ports/cmake/src/libmpg123/CMakeLists.txt
++++ b/ports/cmake/src/libmpg123/CMakeLists.txt
+@@ -235,6 +235,13 @@ else()
+     target_sources(${TARGET} PRIVATE ${PLATFORM_SOURCES})
+ endif()
+ 
++if(HAVE_M)
++    string(APPEND LIBMPG123_LIBS " -lm")
++endif()
++if(WANT_WIN32_UNICODE)
++    string(APPEND LIBMPG123_LIBS " -lshlwapi")
++endif()
++set(LIBMPG123_LIBS "${LIBMPG123_LIBS}" PARENT_SCOPE)
+ target_link_libraries(${TARGET} PRIVATE
+     $<$<BOOL:${HAVE_M}>:m>
+     $<$<BOOL:${WANT_WIN32_UNICODE}>:shlwapi>)
+diff --git a/ports/cmake/src/libout123/CMakeLists.txt b/ports/cmake/src/libout123/CMakeLists.txt
+index d0fe358..cd800d8 100644
+--- a/ports/cmake/src/libout123/CMakeLists.txt
++++ b/ports/cmake/src/libout123/CMakeLists.txt
+@@ -18,6 +18,13 @@ add_library(${TARGET}
+ 
+ set_target_properties(${TARGET} PROPERTIES OUTPUT_NAME out123)
+ 
++if(HAVE_UNIX_DL)
++    string(APPEND LIBOUT123_LIBS " -ldl")
++endif()
++if(WANT_WIN32_UNICODE)
++    string(APPEND LIBOUT123_LIBS " -lshlwapi")
++endif()
++set(LIBOUT123_LIBS "${LIBOUT123_LIBS}" PARENT_SCOPE)
+ target_link_libraries(${TARGET} PRIVATE
+     $<TARGET_NAME_IF_EXISTS:defaultmodule>
+     $<$<BOOL:${HAVE_UNIX_DL}>:dl>
+diff --git a/ports/cmake/src/libsyn123/CMakeLists.txt b/ports/cmake/src/libsyn123/CMakeLists.txt
+index 6edb5b7..0b660f5 100644
+--- a/ports/cmake/src/libsyn123/CMakeLists.txt
++++ b/ports/cmake/src/libsyn123/CMakeLists.txt
+@@ -19,6 +19,13 @@ target_include_directories(${TARGET} INTERFACE
+     "$<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}>"
+     "$<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>")
+ 
++if(HAVE_M)
++    string(APPEND LIBSYN123_LIBS " -lm")
++endif()
++if(WANT_WIN32_UNICODE)
++    string(APPEND LIBSYN123_LIBS " -lshlwapi")
++endif()
++set(LIBSYN123_LIBS "${LIBSYN123_LIBS}" PARENT_SCOPE)
+ target_link_libraries(${TARGET} PRIVATE
+     $<$<BOOL:${HAVE_M}>:m>
+     $<$<BOOL:${WANT_WIN32_UNICODE}>:shlwapi>)

--- a/ports/mpg123/portfile.cmake
+++ b/ports/mpg123/portfile.cmake
@@ -5,7 +5,9 @@ vcpkg_from_sourceforge(
     FILENAME "mpg123-${VERSION}.tar.bz2"
     SHA512 dccb640b0845061811cb41bf145587e7093556d686d49a748232b079659b46284b6cc40db42d14febceac11277c58edf2b69d1b4c46c223829a3d15478e2e26c
     PATCHES
+        fix-dllexport.diff
         have-fpu.diff
+        pkgconfig.diff
 )
 
 if(VCPKG_TARGET_IS_WINDOWS AND NOT VCPKG_TARGET_IS_MINGW)

--- a/ports/mpg123/portfile.cmake
+++ b/ports/mpg123/portfile.cmake
@@ -29,6 +29,8 @@ vcpkg_cmake_configure(
         ${options}
     MAYBE_UNUSED_VARIABLES
         BUILD_PROGRAMS
+        CMAKE_DISABLE_FIND_PACKAGE_ALSA
+        CMAKE_DISABLE_FIND_PACKAGE_PkgConfig
 )
 vcpkg_cmake_install()
 vcpkg_copy_pdbs()

--- a/ports/mpg123/vcpkg.json
+++ b/ports/mpg123/vcpkg.json
@@ -1,6 +1,7 @@
 {
   "name": "mpg123",
   "version": "1.32.9",
+  "port-version": 1,
   "description": "mpg123 is a real time MPEG 1.0/2.0/2.5 audio player/decoder for layers 1, 2 and 3 (MPEG 1.0 layer 3 also known as MP3).",
   "homepage": "https://sourceforge.net/projects/mpg123/",
   "license": "LGPL-2.1-or-later",

--- a/scripts/test_ports/vcpkg-ci-mpg123/portfile.cmake
+++ b/scripts/test_ports/vcpkg-ci-mpg123/portfile.cmake
@@ -1,0 +1,10 @@
+SET(VCPKG_POLICY_EMPTY_PACKAGE enabled)
+
+vcpkg_find_acquire_program(PKGCONFIG)
+
+vcpkg_cmake_configure(
+    SOURCE_PATH "${CURRENT_PORT_DIR}/project"
+    OPTIONS
+        "-DPKG_CONFIG_EXECUTABLE=${PKGCONFIG}"
+)
+vcpkg_cmake_build()

--- a/scripts/test_ports/vcpkg-ci-mpg123/project/CMakeLists.txt
+++ b/scripts/test_ports/vcpkg-ci-mpg123/project/CMakeLists.txt
@@ -1,0 +1,14 @@
+cmake_minimum_required(VERSION 3.30)
+
+project(vcpkg-ci-mpg123 C)
+
+find_package(mpg123 REQUIRED)
+
+add_executable(main main.c)
+target_link_libraries(main PRIVATE MPG123::libmpg123)
+
+find_package(PkgConfig REQUIRED)
+pkg_check_modules(MPG123_PC REQUIRED IMPORTED_TARGET libmpg123)
+
+add_executable(main-pc main.c)
+target_link_libraries(main-pc PRIVATE PkgConfig::MPG123_PC)

--- a/scripts/test_ports/vcpkg-ci-mpg123/project/main.c
+++ b/scripts/test_ports/vcpkg-ci-mpg123/project/main.c
@@ -1,0 +1,11 @@
+#include <mpg123.h>
+
+int main()
+{
+	mpg123_handle *m = mpg123_new(NULL, NULL);
+	mpg123_open(m, "vcpkg");
+	mpg123_scan(m);
+	mpg123_close(m);
+	mpg123_delete(m);
+    return 0;
+}

--- a/scripts/test_ports/vcpkg-ci-mpg123/vcpkg.json
+++ b/scripts/test_ports/vcpkg-ci-mpg123/vcpkg.json
@@ -1,0 +1,12 @@
+{
+  "name": "vcpkg-ci-mpg123",
+  "version-string": "ci",
+  "description": "Validates mpg123",
+  "dependencies": [
+    "mpg123",
+    {
+      "name": "vcpkg-cmake",
+      "host": true
+    }
+  ]
+}

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -6098,7 +6098,7 @@
     },
     "mpg123": {
       "baseline": "1.32.9",
-      "port-version": 0
+      "port-version": 1
     },
     "mpi": {
       "baseline": "1",

--- a/versions/m-/mpg123.json
+++ b/versions/m-/mpg123.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "bc32f5fff20c09b640b752e86cf0562742a43787",
+      "git-tree": "0c11d8d8818fece4a1fb1de93e5423a078f6c29e",
       "version": "1.32.9",
       "port-version": 1
     },

--- a/versions/m-/mpg123.json
+++ b/versions/m-/mpg123.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "bc32f5fff20c09b640b752e86cf0562742a43787",
+      "version": "1.32.9",
+      "port-version": 1
+    },
+    {
       "git-tree": "24f943a44e7512101017022557b2291d5ef209be",
       "version": "1.32.9",
       "port-version": 0

